common4j/src/main/com/microsoft/identity/common/java/cache/NameValueStorageBrokerApplicationMetadataCache.java

```diff
@@ -34,8 +34,10 @@
 import java.util.Set;
 
 import edu.umd.cs.findbugs.annotations.Nullable;
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
 import lombok.NonNull;
 
+@SuppressFBWarnings(value = "RCN_REDUNDANT_NULLCHECK_WOULD_HAVE_BEEN_A_NPE", justification = "Lombok inserts more null checks than we need")
```

Comment:
Can you put the `RCN_REDUNDANT_NULLCHECK_WOULD_HAVE_BEEN_A_NPE` in the `SpotbugsWarning` constants file?